src/parser/core/modules/AoE.js

```diff
@@ -155,22 +155,29 @@ export default class AoE extends Module {
 				const key = `${event.targetID}-${event.targetInstance}`
 				if (carry[key]) {
 					carry[key].times++
+					carry[key].amount += event.amount
```

Comment:
Property is designed to sum all hits (or heals, but I mostly cared about hits on the aoedamage prop) so that there's the same property with either `damage` or `aoedamage` to check if the attack hit.  Per comments further down, I don't know if this is really the right property to be checking for did the attack miss / hit only invuln targets, though I'm not aware of any current content where this wouldn't be a functional thing to check.

Property is being used by the event.amount === 0 check for DRK resource simulation (and that is specifically what I added this for).

Reply:
Missed that usage, my bad.

And looks like I'm a numpty (what a surprise), and forgot to fill in the `HitType` enum that I reverse engineered out of fflogs a few months back. I'll add it in to be used, but in the short term, have a dump:

```ts
export enum HitType {
	MISS = 0,
	NORMAL = 1,
	CRITICAL = 2,
	BLOCK = 4,
	DODGE = 7,
	// PARRY = 8, // Seems to be used for other games where parry is a missType?
	DEFLECT = 9,
	IMMUNE = 10,
	MISFIRE = 11,
	REFLECT = 12,
	EVADE = 13,
	RESIST = 14,
	// TODO: Tentative:
	PARRY = 20,
}
```

This mapping seems to be shared across all XYZLogs sites, so several of them are unused by ff.

Here's the closest mapping I could find:

```ts
// Map FFLogs hit types to the game equivalent. Partial 'cus fflogs has a lot of whack shit that XIV doesn't even produce.
const sourceHitType: Partial<Record<Fflogs.HitType, Parser.SourceModifier>> = {
	[Fflogs.HitType.MISS]: Parser.SourceModifier.MISS,
	[Fflogs.HitType.CRITICAL]: Parser.SourceModifier.CRITICAL,
	// Marking dodge as miss 'cus it seems to be mis-used as such on fflogs
	[Fflogs.HitType.DODGE]: Parser.SourceModifier.MISS,
}
const targetHitType: Partial<Record<Fflogs.HitType, Parser.TargetModifier>> = {
	[Fflogs.HitType.BLOCK]: Parser.TargetModifier.BLOCK,
	[Fflogs.HitType.PARRY]: Parser.TargetModifier.PARRY,
}
```

(sorry for the dump, but hopefully it's helpful. I'll try to get some of this mapped across)